docs/types/mapping-types.rst

```diff
@@ -26,6 +26,7 @@ are allowed for state variables, as storage reference types
 in functions, or as parameters for library functions.
 They cannot be used as parameters or return parameters
 of contract functions that are publicly visible.
+These restrictions are also true for arrays and structs that contain mappings.
 
```

Comment:
@chriseth I could not find any reference to "skipping" mappings. I tried a grep for skip, as well reading the documentation for struct.